heal/vlmd/README.md

```diff
@@ -0,0 +1,33 @@
+# VLMD methods
+
+## VLMD validation
+
+This module validates VLMD data dictionaries against stored schemas.
+
+The `vlmd_validate()` method raises a `jsonschema.ValidationError` for an invalid input file.
+
+Example validation code:
+
+```
+from jsonschema import ValidationError
+
+from heal.vlmd import vlmd_validate
+
+input_file = "vlmd_dd.json"
+try:
+    vlmd_validate(input_file)
+except ValidationError as e:
+    # handle validation error
+
+```
+
+### Adding new validators
```

Comment:
this is great to start with!
in the future when we also got the extractors ready, maybe we should consider put this section into a dedicated dev doc, with more details and some examples, to faciliate development